### DISCUSSION TITLE
Stream/Game Title not working from official Twitch by Elgato plugin

### DISCUSSION THREAD

**Discussion by TheDjRider** [elgatosf/elgato#39] [Help]
Hello, 
long ago maybe 4 months Twitch updated their API and most ways to update Stream Title or Game stopped working. So Elgato's plugin is still broken at this time. The option called "**_Stream/Game Title_**" works only if you have both fields written.

Example: 

This case works fine:
![Changing both Title and Game](https://i.imgur.com/Aw5hNig.png)

This also works fine:
![Changing only Title](https://i.imgur.com/abN8lsy.png)

But in this case, the deck shows an exclamation mark on pressing
![Chaging only game](https://i.imgur.com/YsaYwBK.png)

Long ago I found in the old Reddit thread post about only the title and probably it got fixed only half. 
I hope also the games will be fixed soon because I mainly use the ability to change my games as there isn't an option to keep the title the same as what is right now.

**Comment by CSSully** [comment]
I am also still having this same issue and would love to get it resolved.

**Comment by TheDjRider** [comment]
It's great how bugs are just skipped and ignored, with no answer or commitment to say that will not gonna be fixed.

**Comment by ChekTek** [comment]
Hi @TheDjRider! 

This will be resolved with the next release of the Twitch plugin along with some additional improvements! In the future, please contact [Elgato Support](https://help.elgato.com/hc/en-us/requests/new) for product issues as the message can be communicated more directly with the appropriate team. This forum is used for helping makers building with the Stream Deck SDK.

Thanks for reporting the issue!

Best,
Zack
